safe-global/web-core
# fix: Safe creation issues

## What it solves

Resolves #1170
Resolves #1177 
Resolves #1179

## How this PR fixes it

- Navigates the user to `/welcome` when they cancel safe creation
- Reuses `NameInput` to add 50 character limit
- Changes the fill color of sidebar icons to `currentColor` instead of a hard-coded value to work in dark mode.

## How to test it

1. Open the safe
2. Create a new safe
3. Try to add a name longer than 50 characters
4. Observe an error being shown
5. Cancel safe creation
6. Observe being navigated to /welcome
7. Finish the creation flow
8. Observe the modal icons being visible in dark mode

## Screenshots
<img width="641" alt="Screenshot 2022-11-16 at 17 37 32" src="https://user-images.githubusercontent.com/5880855/202239741-d3cc4f92-0b85-44e5-a566-226ad20f4c7e.png">

<img width="622" alt="Screenshot 2022-11-16 at 17 33 06" src="https://user-images.githubusercontent.com/5880855/202239637-93bbdcd1-210c-431b-8d04-46b460bc87b3.png">
